tests/integration/lfs_view_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package integration
+
+import (
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/tests"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// check that a markup stored in LFS renders its content but is flagged with "Stored in Git LFS"
+func TestLFSRenderMarkup(t *testing.T) {
+	defer tests.PrepareTestEnv(t)()
+
+	session := loginUser(t, "user2")
+
+	req := NewRequest(t, "GET", "/user2/lfs/src/branch/master/CONTRIBUTING.md")
+	resp := session.MakeRequest(t, req, http.StatusOK)
+
+	doc := NewHTMLParser(t, resp.Body).doc
+
+	fileInfo := doc.Find("div.file-info-entry").First().Text()
+	assert.Contains(t, fileInfo, "Stored with Git LFS")
```

Comment:
I wrote the english here, but this string is internationalized. Are we guaranteed that the tests always run in english?